### PR TITLE
Entitlement Management - AP move to new catalog

### PR DESCRIPTION
**Pull request by fsheik**

### CODE REVIEW

File: api-reference/beta/api/accesspackage-movetocatalog.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the parameters.
+
+The following table shows the parameters that can be used with this action.
+
+|Parameter|Type|Description|
+|:---|:---|:---|
+|catalogId|String|ID of the Catalog to which the access package will be moved to.|
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackage-movetocatalog.md:49]: nit
```suggestion
|catalogId|String|ID of the catalog to which the access package will be moved.|
```

File: api-reference/beta/api/accesspackage-movetocatalog.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+### Request
+The following is an example of a request.
+<!-- {
+  "blockType": "request",
+  "name": "accesspackagethis.movetocatalog"
+}
+-->
+``` http
+POST https://graph.microsoft.com/beta/identityGovernance/entitlementManagement/accessPackages/{accessPackageId}/moveToCatalog
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackage-movetocatalog.md:67]: Replace "{accessPackageId}" with real looking ID.

File: api-reference/beta/api/accesspackage-movetocatalog.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+Content-Type: application/json
+Content-length: 57
```
Comment (FaithOmbongi) [inline api-reference/beta/api/accesspackage-movetocatalog.md:69]:
```suggestion
```

File: api-reference/beta/toc.yml [hunk 4]
```diff
@@ -13106,6 +13106,8 @@ items:
               href: api/accesspackage-delete-incompatiblegroup.md
             - name: List access packages incompatible with
               href: api/accesspackage-list-accesspackagesincompatiblewith.md
+            - name: Move
```
Comment (FaithOmbongi) [inline api-reference/beta/toc.yml:13109]: nit
```suggestion
            - name: Move to catalog
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you, @fsheik, for your work on this PR. Approved with suggestions for minor fixes.

File: api-reference/beta/api/accesspackage-movetocatalog.md (hunk 2 above)
Comment (markwahl-msft) [reply api-reference/beta/api/accesspackage-movetocatalog.md]: updated

**Pull request merged**